docs/development/api/classes/plugin_lib.Headlamp.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Class: Headlamp"
+linkTitle: "Headlamp"
+slug: "plugin_lib.Headlamp"
+---
+
+[plugin/lib](../modules/plugin_lib.md).Headlamp
+
+This class is a more convenient way for plugins to call registerPlugin in order to register
+themselves.
+
+## Constructors
+
+### constructor
+
+• **new Headlamp**()
+
```

Comment:
We only have a method in this class and it's a static one. Should we still mention the constructor here?